### PR TITLE
Added animations to the blog section and videos section

### PR DESCRIPTION
**Pull request by supunTE**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request
I make this new PR for this PR https://github.com/fossasia/gci17.fossasia.org/pull/384 was merged and reverted. So I can't make new changes into that.  
Added animation for the blog section and videos section as issue #363 for point Add animations to the blogs section and Add animations to the videos section. 
https://stedirisuriya.github.io/gci17.fossasia.org/

### CODE REVIEW

**Comment by supunTE** [comment]
It works on mobile devices.
anim
anim2

**Review by akshatnitd — COMMENTED** [review]
The changes in the demo link looks good, but you need to sync your forked repo with the original repo, as other changes are also reflecting in your PR.

File: css/blog.css [hunk 1]
```diff
@@ -0,0 +1,104 @@
+@import url(https://fonts.google.com/specimen/Raleway);
```
Comment (abishekvashok) [inline css/blog.css]: Can we use the built in fonts we are using throughout the site pls?

**Comment by abishekvashok** [comment]
Also properly rebase your fork.

File: css/blog.css (hunk 1 above)
Comment (supunTE) [reply css/blog.css]: What's the problem with this? Same fonts. Didn't?

Comment (abishekvashok) [reply css/blog.css]: Can't we go for a single font per the entire website.

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Squash the commits. And follow the @abishekvashok reviews. :sparkles:

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]
Please pull recent changes and rebase.

File: _data/student_projects.yml [hunk 2]
```diff
@@ -14,4 +14,13 @@
 - name: Mochammad Rafi Al Rizkan
   github: https://github.com/kol4k/eKantin
   author: Rafi
-  image: rafi.png
\ No newline at end of file
+<<<<<<< HEAD
```
Comment (abishekvashok) [inline _data/student_projects.yml]: Invalid or bad conflict resolution. Please take in more care while resolving conflicts.

**Review by abishekvashok — CHANGES_REQUESTED** [review]

File: _data/student_projects.yml (hunk 2 above)
Comment (NaviRocker) [inline _data/student_projects.yml]: Remove these lines

**Review by NaviRocker — CHANGES_REQUESTED** [review]
Don't be hurry. Take your Own Time. Don't forget to squash your commits.

File: _data/student_projects.yml [hunk 3]
```diff
@@ -14,4 +14,13 @@
...
+  image: rafi.png
+
+- name: GCI FOSSASIA Poster
+  github: https://github.com/yashkumarverma
+  author: Yash Kumar Verma
+  image: yashkumarverma-gciposter.jpg
+=======
```
Comment (NaviRocker) [inline _data/student_projects.yml]: Another one

**Review by NaviRocker — CHANGES_REQUESTED** [review]

**Review by abishekvashok — APPROVED** [review]
LGTM

**Comment by lunaticmonk** [comment]
Please squash the commits

**Review by NaviRocker — APPROVED** [review]
LGTM

File: index.html [hunk 4]
```diff
@@ -378,8 +378,13 @@ <h4 class="uppercase mb16">Blogs</h4>
 						<div class="image-tile outer-title text-center">
 							<img data-src="img/blog/{{blog.img}}" height="240" width="240">
 							<div class="title mb16">
-								<h5 class="uppercase mb0">{{blog.title}}</h5>
-								<span>{{blog.author_name}}</span>
+								<h5 class="uppercase mb0">
```
Comment (vaibhavsingh97) [inline index.html]: Formatting needed

File: index.html [hunk 5]
```diff
@@ -412,8 +417,12 @@ <h4 class="uppercase mb16">Videos</h4>
 						<div class="image-tile outer-title text-center">
 							<img data-src="img/video/{{video.img}}" height="240" width="240">
 							<div class="title mb16">
-								<h5 class="uppercase mb0">{{video.title}}</h5>
-								<span>{{video.author_name}}</span>
+									<h5 class="uppercase mb0">
```
Comment (vaibhavsingh97) [inline index.html]: same

File: css/blog.css [hunk 6]
```diff
@@ -0,0 +1,100 @@
+
+.content img{
```
Comment (vaibhavsingh97) [inline css/blog.css:7]: Please do necessary formatting

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

**Comment by vaibhavsingh97** [comment]
Please resolve conflicts as well. Thanks

**Comment by supunTE** [comment]
OK

File: css/blog.css [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+.content .titletext {
+    color:rgb(58, 55, 55);
+    font-size:20px;
+    position:absolute;
+    left:220px;
+    top:50px;
+    opacity:0;
+
+-webkit-transition: all 800ms cubic-bezier(0.000, 0.440, 0.000, 0.795);
```
Comment (vaibhavsingh97) [inline css/blog.css:40]: It's still not indented

File: css/blog.css [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+    top:50px;
+    opacity:0;
+
+-webkit-transition: all 800ms cubic-bezier(0.000, 0.440, 0.000, 0.795); 
+   -moz-transition: all 800ms cubic-bezier(0.000, 0.440, 0.000, 0.795); 
+    -ms-transition: all 800ms cubic-bezier(0.000, 0.440, 0.000, 0.795); 
+     -o-transition: all 800ms cubic-bezier(0.000, 0.440, 0.000, 0.795); 
+        transition: all 800ms cubic-bezier(0.000, 0.440, 0.000, 0.795);
+
+-webkit-transition-timing-function: cubic-bezier(0.000, 0.440, 0.000, 0.795);
```
Comment (vaibhavsingh97) [inline css/blog.css:46]: same

File: css/blog.css [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+
+.content .nameofauthor {
+    color:rgb(0, 0, 0);
+    font-size:15px;
+    position:absolute;
+    left:-1000px;
+    top:200px;
+    opacity:0;
+
+-webkit-transition: all 800ms cubic-bezier(0.000, 0.890, 0.630, 0.980);
```
Comment (vaibhavsingh97) [inline css/blog.css]: same

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

**Review by isuruAb — COMMENTED** [review]
Please resolve conflicts

File: _data/student_projects.yml [hunk 10]
```diff
@@ -14,4 +14,4 @@
 - name: Mochammad Rafi Al Rizkan
   github: https://github.com/kol4k/eKantin
   author: Rafi
-  image: rafi.png
\ No newline at end of file
+  image: rafi.png
```
Comment (vaibhavsingh97) [inline _data/student_projects.yml:17]: Add one blank line

File: css/blog.css [hunk 11]
```diff
@@ -0,0 +1,98 @@
...
+    z-index: -1;
+    background: #000;
+    position: absolute;
+    box-shadow: 100px 200px 300px #000;
+    -moz-transform: scale(1) rotate(5deg) translateX(0px) translateY(0px) skewX(0deg) skewY(0deg);
+    -webkit-transform: scale(1) rotate(5deg) translateX(0px) translateY(0px) skewX(0deg) skewY(0deg);
+    -o-transform: scale(1) rotate(5deg) translateX(0px) translateY(0px) skewX(0deg) skewY(0deg);
+    -ms-transform: scale(1) rotate(5deg) translateX(0px) translateY(0px) skewX(0deg) skewY(0deg);
+    transform: scale(1) rotate(5deg) translateX(0px) translateY(0px) skewX(0deg) skewY(0deg);
+}
```
Comment (vaibhavsingh97) [inline css/blog.css:106]: Add one blank line

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

**Review by abhinavk96 — APPROVED** [review]

**Comment by supunTE** [comment]
there is big issue with this So I close this

**Pull request closed without merging**